codechain/codechain.yml

```diff
@@ -133,21 +133,14 @@ args:
         long: enable-devel-api
         help: Enable the RPC's devel APIs
         takes_value: false
-    - no-miner:
```

Comment:
For the backward compatibility, please keep this CLI and show the warning message that this option is deprecated.

Reply:
Let's remove it on 2.0 release

Reply:
I kept the `no-miner` CLI. And wanted to add a message using `cwarn!` macro but the CLI options are processed before building `logger`, I just added a warning message using `println!`